ui/pages/requests/controller.go

```diff
@@ -757,6 +761,49 @@ func (c *Controller) viewCollection(id string) {
 	c.view.OpenCollectionContainer(col)
 }
 
+func (c *Controller) duplicateCollection(id string) {
+	col := c.model.GetCollection(id)
+	if col == nil {
+		return
+	}
+
+	colClone := col.Clone()
+	colClone.MetaData.Name += " (copy)"
+
+	dirPath, err := c.repo.GetNewCollectionDir(colClone.MetaData.Name)
+	if err != nil {
+		fmt.Println("failed to get new collection dir", err)
+		return
```

Comment:
Why not returning a wrapped error by changing the method signature?

Reply:
K k ow it's out of the PR scope. But I'm surprised by the `fmt.Println ("whatever", err) + return pattern` I see a lot in your code

Reply:
you right, the issue at moment is that I don't have a structure to show user this kind of errors. something like a modal widget. 
once I fix that, I'll change this.

Reply:
Modal or not, I would expect the error to be returned and forwarded to one method where errors will be handled only once